index.coffee

```diff
@@ -1,157 +1,173 @@
 ChildProcess = require 'child_process'
-fs = require 'fs'
 asar = require 'asar'
-path = require 'path'
 temp = require 'temp'
 _ = require 'underscore'
+jetpack = require 'fs-jetpack'
 
 temp.track()
 
-module.exports = (grunt) ->
+consoleLogger =
+  error: (args...) ->
+    console.error.apply(null, args)
```

Comment:
This could be `console.error(args...)` to use the CoffeeScript way, and similarly the ones below.